README.md

```diff
@@ -76,68 +90,86 @@ forum.description = "Ruby programming tips."
+Start the server, and go to <http://localhost/3000/forums>. You see the list of forums -- but no descriptions.  This is because the views were generated when the forums table did not have a description column.  We now need to fix those views.
 
-There are only two kinds of embedded Ruby blocks.  The ones that start ```<% ``` are executed, but do not result in anything added to the HTML.  These are for conditional statements and loops.  The blocks that start ```<%= ``` do generate output.  The result of the Ruby expression is inserted into the HTML. You can put all kinds of Ruby code into an erb file, but this is unwise.  You want to keep your logic in the controller or model, except for a minimum as needed to customize the view.  Note also the line that says
-```
+Edit `app/views/forums/index.html.erb`.  Again, you see those `<% %>` tags that indicate embedded Ruby.  Now, the first thing to understand about those sections is that no Ruby code is sent to the browser.  The browser can do nothing with Ruby.  The Ruby code is executed to generate plain HTML, which is what is sent to the browser.  If you go into developer tools in your browser, that's what you'll see.
+
+There are two kinds of embedded Ruby tags to go over in this lesson.  The ones that start with `<%` are executed, but do not result in anything added to the HTML.  These are for conditional statements and loops.  The code that starts with `<%=` does generate output.  The result of the Ruby expression is inserted into the HTML.  You can put all kinds of Ruby code into an ERB file, but this is unwise.  You want to keep your logic in the controller or model, except for a minimum as needed to customize the view.  Note also the line:
+
+```erb
 <% @forums.each do |forum| %>
 ```
-This is Ruby code that generates no output -- except the other statements in the loop may generate output.  The @forums is the instance variable from the index method of the forums controller.  The views you render have access to instance variables from the controller.
 
-So, let's try modifying the index.html.erb line.  Right below the forums div, add the following code:
-```
+This is Ruby code that generates no output by itself -- but the other statements in the loop may generate output.  The `@forums` is the instance variable from the `index` method of the forums controller.  The views you render have access to instance variables from the controller.
+
+So, let's try modifying the `index.html.erb` line.  Right below the forums `div`, add the following code:
+
+```erb
 <div>
   <% 5.times do |i| %>
-  <p><%= "iteration #{i}" %></p>
+    <p><%= "iteration #{i}" %></p>
   <% end %>
 </div>
```

Comment:
As a matter of style, indentation is for readability of the template, and it doesn't matter what the resulting HTML looks like.